e2e-tests/insurance/cypress/e2e/journeys/submit-textarea-fields-with-special-characters/submit-company-textarea-fields-with-special-characters.spec.js

```diff
@@ -0,0 +1,69 @@
+import { INSURANCE_ROUTES } from '../../../../../constants/routes/insurance';
+import { EXPORTER_BUSINESS as EXPORTER_BUSINESS_FIELD_IDS } from '../../../../../constants/field-ids/insurance/business';
+import { field, backLink } from '../../../../../pages/shared';
+import mockStringWithSpecialCharacters from '../../../../../fixtures/string-with-special-characters';
+
+const {
+  ROOT,
+  EXPORTER_BUSINESS: {
+    ALTERNATIVE_TRADING_ADDRESS_ROOT,
+  },
+} = INSURANCE_ROUTES;
+
+const {
+  ALTERNATIVE_TRADING_ADDRESS: {
+    FULL_ADDRESS,
+  },
+} = EXPORTER_BUSINESS_FIELD_IDS;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Textarea fields - `Business` textarea fields should render special characters without character codes after submission', () => {
```

Comment:
should we call the file submit-business-textarea fields or change the context to say company?

Reply:
Great spot, have renamed the file and description to be aligned with "business"